server/src/main/java/com/dut/healthme/controller/ItemController.java

```diff
@@ -0,0 +1,28 @@
+package com.dut.healthme.controller;
+
+import com.dut.healthme.common.model.AbstractResponse;
+import com.dut.healthme.dto.request.LoginRequest;
+import com.dut.healthme.dto.response.ListRecommendResponse;
+import com.dut.healthme.service.ItemService;
+import io.lettuce.core.dynamic.annotation.Param;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("Item")
```

Comment:
Để đồng bộ với hệ thống thì nên sửa lại thành như sau nha
```suggestion
@RequestMapping("/v1/item")
```